creator/creator.go

```diff
@@ -291,10 +291,11 @@ func (c *Creator) initContext() {
 }
 
 // NewPage adds a new Page to the Creator and sets as the active Page.
-func (c *Creator) NewPage() {
+func (c *Creator) NewPage() *model.PdfPage {
 	page := c.newPage()
 	c.pages = append(c.pages, page)
 	c.context.Page++
+	return page
```

Comment:
In my use case, I wanted to add annotations but wasn't able to do so because I didn't got access to the page.

Reply:
Great, after some discussion we decided this is useful.  I've often wanted to access the page of the creator.
The reason we did not allow it, is that it could potentially conflict with the creator's operators on the page, although I cannot think exactly of a way that could happen.